examples/frameworks/strands_demo/src/nat_strands_demo/register.py

```
@@ -0,0 +1,104 @@
+from pydantic import Field
+
+from nat.builder.builder import Builder
+from nat.builder.framework_enum import LLMFrameworkEnum
+from nat.builder.function_info import FunctionInfo
+from nat.cli.register_workflow import register_function
+from nat.data_models.component_ref import FunctionRef
+from nat.data_models.component_ref import LLMRef
+from nat.data_models.function import FunctionBaseConfig
+from nat.data_models.optimizable import OptimizableMixin
+
+from . import ping_tool  # noqa: F401, pylint: disable=unused-import
```

Ruff (0.14.3)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)